Assets/js/scripts.js

```diff
@@ -62,6 +62,16 @@ const goto = (section) => {
   menuClose();
 };
 
+const modalClose = (opc = false) => {
```

Comment:
- [ ] Please add the `link to live version` and `link to the project source code` attributes in each project object in the projects array.
- [ ] Also update the href value in the `see live` and `see source` links with values of the `link to live version` and `link to the project source code` attributes.

Reply:
in the new commit, I repair this issue, thanks for the feedback